### PR TITLE
Lesson 2

### PR DESCRIPTION
**Pull request by Irina-pi**

### CODE REVIEW

File: Hello/about.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      Ну а наиболее творческое направление я увидела у
+      <em>Front-end</em> разработчиков. Сюда и направилась.
+    </p>
+    <h3>Напишу немного о себе</h3>
+    <p>
+      Родилась я 29 лет назад. Школа, медицинский университет, интернатура,
+      работа. На данный момнет я работаю в
+      <del>поликлинике</del>
+      <abbr title="Гомельская областная инфекционная клиническая больница"
+        >ГОИКБ</abbr
```
Comment (5tronciy) [inline Hello/about.html]: Hereinafter tags shouldn't be visibly broken.
Check the prettier's settings, better set it to default.

File: Hello/about.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    </p>
+    <small
+      >Моя рабочая почта:<a href="mailto:rabotairinabocharova@yandex.by"
+        >rabotairinabocharova@yandex.by</a
+      ></small
+    >
+    <hr />
+    <a href="../index.html">Home</a>
+  </body>
+</html>
```
Comment (5tronciy) [inline Hello/about.html:48]: Used 2 types of headers only.
No quotes, link to github.

**Review by 5tronciy — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 .idea
+node_modules
```
Comment (5tronciy) [inline .gitignore:2]: node_modules and node_modules/ are almost the same.

**Review by 5tronciy — CHANGES_REQUESTED** [review]

File: Hello/about.html (hunk 2 above)
Comment (Irina-pi) [reply Hello/about.html:48]: I corrected already.

File: .gitignore (hunk 3 above)
Comment (Irina-pi) [reply .gitignore:2]: sorry, i do not understend.

File: Hello/about.html (hunk 1 above)
Comment (Irina-pi) [reply Hello/about.html]: repeated attempts to correct the error did not give results.

File: .gitignore (hunk 3 above)
Comment (5tronciy) [reply .gitignore:2]: Read this article: [gitignore](https://www.atlassian.com/ru/git/tutorials/saving-changes/gitignore), and look at the task carefully.

**Comment by 5tronciy** [comment]
Looks good now.

**Pull request merged**